packages/teleterm/src/ui/ModalsHost/modals/UsageData/UsageData.tsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2022 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import DialogConfirmation, {
+  DialogContent,
+  DialogFooter,
+  DialogHeader,
+} from 'design/DialogConfirmation';
+import { ButtonIcon, ButtonPrimary, ButtonSecondary, Text } from 'design';
+import { Close } from 'design/Icon';
+
+interface UsageDataProps {
+  onCancel(): void;
+
+  onAllow(): void;
+
+  onDecline(): void;
+}
+
+export function UsageData(props: UsageDataProps) {
+  return (
+    <DialogConfirmation
+      open={true}
+      onClose={props.onCancel}
+      dialogCss={() => ({
+        maxWidth: '400px',
+        width: '100%',
+      })}
+    >
+      <form
+        onSubmit={e => {
+          e.preventDefault();
+          props.onAllow();
+        }}
+      >
+        <DialogHeader
+          justifyContent="space-between"
+          mb={0}
+          alignItems="baseline"
+        >
+          <Text typography="h4" bold>
+            Anonymous usage data
+          </Text>
+          <ButtonIcon
+            type="button"
+            onClick={props.onCancel}
+            color="text.secondary"
+          >
+            <Close fontSize={5} />
+          </ButtonIcon>
+        </DialogHeader>
+        <DialogContent mb={4}>
+          <Text typography="body1" color="text.secondary">
+            Do you allow for sending anonymized usage data about Teleport
```

Comment:
This reads a bit weird. I'd suggest something along the lines of
"Do you allow us to collect anonymized usage data...". If you want to keep it as close to what you have now as possible, we should replace "for" with "the", but tbh, both sound a bit odd. 

Also, and this may just be me, but hearing "we are collecting" instead of "we are sending" sounds more like WE are in control of the data instead of just shipping it off elsewhere. idk, maybe I'm over thinking it

Reply:
Thanks for the suggestions guys!
I decided to go with @ravicious version, it sounds less formal :)